packages/blade/src/components/Button/BaseButton/buttonTokens.ts

```diff
@@ -0,0 +1,36 @@
+const typography = {
+  onDesktop: {
+    fonts: {
+      size: {
+        large: 200,
+        medium: 100,
+        small: 75,
+        xsmall: 75,
+      },
+    },
+    lineHeight: {
+      large: 's',
+      medium: 'l',
+      small: 'l',
+      xsmall: 'l',
+    },
+  },
+  onMobile: {
+    fonts: {
+      size: {
+        large: 200,
+        medium: 100,
+        small: 50,
+        xsmall: 50,
+      },
+    },
+    lineHeight: {
+      large: 'm', // this is 24px but we need 20px
+      medium: 'm', // this is 24px but we need 20px
```

Comment:
@saurav12 something to look into?

Reply:
Yes, had to discuss this with @saurav12. I'll bring it up with him during standup